### PR TITLE
En 166 create profile for integration testing

### PR DESCRIPTION
**Pull request by ptylczynski**
New config is copy of dev config with changed DB target to H2. Now it could be used during test with `@ActiveProfiles("integration")`

### CODE REVIEW

File: src/main/resources/application-integration.properties [hunk 1]
```diff
@@ -0,0 +1,42 @@
+transcription.api.url=http://fresenia.man.poznan.pl/tp-api
```
Comment (helinek) [inline src/main/resources/application-integration.properties:1]: I am not sure whether in dev props there are correct URLs but we should use dev tp-api, not the production one

File: src/main/resources/application-integration.properties [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+api.tokenEndpoint=https://auth.europeana.eu/auth/realms/europeana/protocol/openid-connect/token
+# Search API
+ddb.api.key=yVu8QOlMksagoWiamZfkuDDsAgEho6Ozm4haAks2FuzmMGWsXXb1558442839394
+ddb.api.url=https://api.deutsche-digitale-bibliothek.de
+ddb.search.api.url=/search
+ddb.format.api.url=/items/{recordId}/binaries
+ddb.binary.api.url=/binary/
+ddb.api.itemsEndpoint=/items/{recordId}/edm
+#conversion.iiif.server.url=iks-portal.synat.pcss.pl
+conversion.iiif.server.url=rhus-209.man.poznan.pl
```
Comment (helinek) [inline src/main/resources/application-integration.properties:40]: iiif server should be the test instance, not production (rhus-148)

**Review by helinek — CHANGES_REQUESTED** [review]

File: src/main/resources/application-integration.properties (hunk 1 above)
Comment (ptylczynski) [reply src/main/resources/application-integration.properties]: changed

File: src/main/resources/application-integration.properties (hunk 2 above)
Comment (ptylczynski) [reply src/main/resources/application-integration.properties]: changed

**Pull request merged**